.github/pull_request_template.md

```diff
@@ -20,6 +20,7 @@ Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wik
 - [ ] Performed manual testing (Chrome and Firefox if it includes front-end changes)
 - [ ] Extended the README / documentation, if necessary. For user-facing features, consider updating [the user docs](https://github.com/civiform/docs). For "under-the-hood" changes or things more relevant to developers, consider updating [the dev wiki](https://github.com/civiform/civiform/wiki).
 - [ ] Ensured PII wasn't added to any new logs, unless it was guarded by `isDevOrStaging`
+- [ ] Noted in the PR description if any code in this PR was generated by AI.
```

Comment:
Should this be yes or no or text description of which code is generated by AI? I'm thinking in terms of what would be useful to the code reviewer(whether a single line or whole file or whole PR was used to generated by AI).

Reply:
Great question. I think it would be most helpful to note which cod  was generated by AI. I'll update that.